MediaAdminBundle/Resources/public/ecmascript/OpenOrchestra/Application/View/Media/MediaFormView.js

```diff
@@ -41,6 +41,18 @@ class MediaFormView extends mix(AbstractFormView).with(FormViewButtonsMixin)
 
         return this;
     }
+
+    /**
+     * Render a form
+     *
+     * @private
+     */
+    _renderForm() {
+        super._renderForm();
+        $('#page-name', this.$el).html($("input[id*='oo_media_']", this.$el).first().val());
```

Comment:
@alavieille : all my answers in https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/2150